buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/FirebaseLibraryPlugin.java

```diff
@@ -32,6 +31,25 @@ public void apply(Project project) {
         project.getExtensions().create("firebaseLibrary", FirebaseLibraryExtension.class, project);
 
     LibraryExtension android = project.getExtensions().getByType(LibraryExtension.class);
+    android.buildTypes(
+        types ->
+            types
```

Comment:
You've lost the comment here about why you're copying the signing configuration from the debug to release build type. Assuming this is still accurate, maybe add this back above `android.buildTypes`?

```
// In the case of and android library signing config only affects instrumentation test APK.	
// We need it signed with default debug credentials in order for FTL to accept the APK.
```

Reply:
nice catch, added it back